src/Illuminate/Notifications/Messages/MailMessage.php

```diff
@@ -206,6 +206,19 @@ public function attach($file, array $options = [])
         return $this;
     }
 
+    /**
+     * Attach a file to the message based on a given condition.
+     *
+     * @param bool $condition
+     * @param  string $file
+     * @param  array $options
+     * @return $this
+     */
+    public function attachWhen(bool $condition, $file, array $options = [])
```

Comment:
Laravel doesn't allow primitive type-hint.

Reply:
Okay, thanks, I fixed it.